src/Entries/Entry.php

```diff
@@ -665,6 +680,16 @@ public function values()
         return $this->collection()->cascade()->merge($this->originValues());
     }
 
+    public function disableEvents()
+    {
+        $this->withEvents = false;
+    }
+
+    public function enableEvents()
+    {
+        $this->withEvents = true;
+    }
+
```

Comment:
I don't think these methods are necessary right now. Just change the property within `saveQuietly`.